e2e/poms/task1.ts

```diff
@@ -0,0 +1,190 @@
+import { TASK1TEXT } from "../constants/texts/task1texts";
+
+export class Task1 {
+    constructor(private page: Page) {}
+
+    async buildPage() {
+        await this.page.goto(TASK1SELECTOR.urls.home);
+        await this.page.getByTestId(TASK1SELECTOR.testIds.addFormButton).click();
+        await this.page.getByTestId(TASK1SELECTOR.testIds.startFromScratchButton).click();
+    }
+    
+    async buildFor1() {
+        await this.page.getByTestId(TASK1SELECTOR.testIds.addFullNameElement).click();
+        await expect(this.page.getByTestId(TASK1SELECTOR.testIds.fullNamePreviewGroup)).toBeVisible();
+        await this.page.getByTestId(TASK1SELECTOR.testIds.addPhoneNumberElement).click();
+        await expect(this.page.getByTestId(TASK1SELECTOR.testIds.phonePreviewGroup)).toBeVisible();
+
+        await this.page.getByTestId(TASK1SELECTOR.testIds.publishButton).click();
+        const previewPromise = this.page.waitForEvent("popup");
+        await this.page.getByTestId(TASK1SELECTOR.testIds.publishPreviewButton).click();
+        const page2 = await previewPromise;
+        return page2;
+    }
+
+    async submit(page2: Page) {
+        await expect(page2.getByTestId(TASK1SELECTOR.testIds.phoneGroup)).toBeVisible();
+        await expect(page2.getByTestId(TASK1SELECTOR.testIds.fullNameGroup)).toBeVisible();
+        await expect(page2.getByTestId(TASK1SELECTOR.testIds.emailGroup)).toBeVisible();
+        await page2.getByTestId(TASK1SELECTOR.testIds.startOrSubmitButton).click();
+        await expect(page2.getByText(TASK1TEXT.texts.phoneNumberInvalid)).toBeVisible();
+        await expect(page2.getByText(TASK1TEXT.texts.firstNameRequired)).toBeVisible();
+        await expect(page2.getByText(TASK1TEXT.texts.lastNameRequired)).toBeVisible();
+        await expect(page2.getByText(TASK1TEXT.texts.emailRequired)).toBeVisible();
+
+        await page2.getByTestId(TASK1SELECTOR.testIds.emailTextField).fill(TASK1TEXT.texts.email)
+        await page2.getByTestId(TASK1SELECTOR.testIds.firstNameTextField).fill(TASK1TEXT.texts.firstName)
+        await page2.getByTestId(TASK1SELECTOR.testIds.lastNameTextField).fill(TASK1TEXT.texts.lastName)
+        await page2.getByTestId(TASK1SELECTOR.testIds.phoneNumberInputField).fill(TASK1TEXT.texts.phoneNo)
+        await page2.getByTestId(TASK1SELECTOR.testIds.startOrSubmitButton).click();
+        await expect(page2.getByTestId(TASK1SELECTOR.testIds.thankYouPageContent)).toBeVisible();
+
+        await this.page.bringToFront();
+        await this.page.waitForTimeout(TASK1SELECTOR.timeouts.short);
+        await this.page.getByTestId(TASK1SELECTOR.testIds.submissionsTab).click();
+        await expect(this.page.getByText(TASK1TEXT.texts.email)).toBeVisible();
+    }
+
+    async deleteForm() {
+        await this.page.bringToFront();
+        await this.page.locator(`//button[@data-testid='${TASK1SELECTOR.testIds.neetoMoleculesMenuButton}']`).click()
```

Comment:
Always try to use data-cy labels. In rare-cases it is not available and you have to use page.locator, there is a better method.
```suggestion
        // Make TASK1SELECTOR.testIds.neetoMoleculesMenuButton: "[data-testId='neeto-molecules-menu-button']"
        await this.page.locator(TASK1SELECTOR.testIds.neetoMoleculesMenuButton).click()
```